pkgs/development/compilers/gcc/6/default.nix

```diff
@@ -317,7 +317,9 @@ stdenv.mkDerivation ({
 
   NIX_LDFLAGS = stdenv.lib.optionalString  hostPlatform.isSunOS "-lm -ldl";
 
-  preConfigure = stdenv.lib.optionalString (hostPlatform.isSunOS && hostPlatform.is64bit) ''
+  preConfigure = ''
+    patchShebangs configure
```

Comment:
Why is this change to gcc6 necessary?

Reply:
Oh, its not. That was the fix for https://github.com/NixOS/nixpkgs/issues/36754 that I forgot to remove before commiting. Good catch.

I removed it and force pushed.